.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false
 
 language: go
 go:
-- 1.5
```

Comment:
Ahh, good point, I think we want to use 1.7.x to keep things pinned to a release.